.travis.yml

```diff
@@ -1,28 +1,25 @@
 sudo: false
 language: node_js
-cache:
-  directories:
-    - node_modules
 notifications:
   email: false
 node_js:
   - 4
   - 6
 matrix:
-    fast_finish: true
-    allow_failures:
+  fast_finish: true
+  allow_failures: ~
```

Comment:
can we do these changes another way then? There are 3 different places in this file where the contents of the file change without having any actual effect on the tests (this one, removing the quotes a few lines down, and removing the trailing newline). it would be nice to keep these PRs to only changes that actually modify how our tests run to keep things clean.

I've had success using bash scripts to do this sort of thing in the past, although they are kinda a pain to set up.